addNRemove.test.js

```diff
@@ -0,0 +1,21 @@
+const addToDo = require('./utils/addTest.js');
+const removeTest = require('./utils/removeTest.js');
+const getFilledStorage = require('./__mocks__/filledTaskList.js');
+const getLocalStorage = require('./__mocks__/taskList.js');
+
+const localStorageList = getLocalStorage();
+const filledTaskList = getFilledStorage();
+
```

Comment:
- [ ] Great job with the testing and you are using a newer version of jest, as such localStorage is mocked automatically. But you missed q point as per the project requirements. 
- `Make sure you group your tests using the describe() method.`

I cannot find any `describe() `method in your test files. Kindly get this fixed.